src/helpers/config.ts

```diff
@@ -294,13 +327,29 @@ export const allCoinMetas: Record<string, CoinMetadata> = {
     value: 'tarbeth',
     ApiKeyProvider: 'arbiscan.io',
   },
+  tarbethToken: {
+    Title: 'TARBETH TOKEN',
+    Description: 'Testnet Arbitrum Token',
+    Icon: 'arbeth',
+    value: 'tarbethToken',
+    minGasLimit: '400,000',
+    defaultGasLimit: '1,000,000',
+  },
```

Comment:
defaultGasLimitNum needs to be added here as well?

Reply:
addressed